### PR TITLE
Feature/administrationmenu

### PR DESCRIPTION
**Pull request by dward2nd**
- Old setting views related to Administration were moved from basxconnect/core/views/settings_views to basxconnect/core/views/admin_views
- In the Maintenance setting view, a package version list was added.

### CODE REVIEW

File: basxconnect/core/views/admin_views.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+@aslayout
+def maintenancesettings(request):
+    ret = layout.grid.Grid(R(C(hg.H3(_("Maintenance")))), gutter=False)
+
+    package_current = []
+    package_latest = []
+
+    for package_name in PACKAGE_NAMES:
+        current_version = pkg_resources.get_distribution(package_name).version
+        newer_version = "unable to load"
```
Comment (saemideluxe) [inline basxconnect/core/views/admin_views.py:33]: Make this a translatable string please.

File: basxconnect/core/views/admin_views.py [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+                            header="Package",
+                            cell=hg.DIV(hg.C("row.package_name")),
+                        ),
+                        DataTableColumn(
+                            header="Current",
+                            cell=hg.DIV(hg.C("row.package_current")),
+                        ),
+                        DataTableColumn(
+                            header="Latest",
+                            cell=(hg.DIV(hg.C("row.package_latest"))),
```
Comment (saemideluxe) [inline basxconnect/core/views/admin_views.py:58]: Make header entries translatable strings.

**Review by saemideluxe — CHANGES_REQUESTED** [review]
Okay, this looks very good. Just make the changes for translatable strings and fix the warnings for flake8. I updated the flake8 config, there will be two warnings.

You can check it on your own machine with ```pip install flake8 flake8-black``` and then just run ```flake8``` inside the basxconnect repository or you see if pycharm can detect our flake8 configuration (inside the file ```.flake8```).

**Comment by dward2nd** [comment]
I resolved the visible strings to be translatable as suggested.

**Comment by saemideluxe** [comment]
Okay, thanks. Did you also see my notes on flake8 linting? If you check details of the basxconnect-ci check, you will see that the flake8 linting fails with two errors:

```bash
./basxconnect/core/views/person/person_modals_views.py:1:1: F401 'bread' imported but unused
./basxconnect/core/layouts/settings_layout.py:2:1: F401 'pkg_resources' imported but unused
```

**Pull request merged**